packages/admin/cms-admin/src/dam/DataGrid/thumbnail/DamThumbnail.tsx

```diff
@@ -37,10 +47,38 @@ interface DamThumbnailProps {
 }
 
 export const DamThumbnail = ({ asset }: DamThumbnailProps): React.ReactElement => {
+    const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+
+    const open = Boolean(anchorEl);
+
+    const handleMouseOver = (event: React.MouseEvent<HTMLElement>) => {
+        setAnchorEl(event.currentTarget);
+    };
+    const handleMouseLeave = () => {
+        setAnchorEl(null);
+    };
+
     let thumbnail;
     if (asset.__typename === "DamFile") {
         if (asset.mimetype.startsWith("image/") && asset.image && asset.image.thumbnailUrl) {
-            thumbnail = <ImageThumbnail src={asset.image.thumbnailUrl} />;
+            const id = open ? asset.fileUrl : undefined;
+            thumbnail = (
+                <>
+                    <ImageThumbnail
+                        aria-describedby={id}
+                        onMouseOver={handleMouseOver}
+                        onMouseLeave={handleMouseLeave}
+                        src={asset.image.thumbnailUrl}
+                    />
+                    <Popper open={open} anchorEl={anchorEl} placement="auto-end" onResize={undefined} onResizeCapture={undefined} transition>
+                        {({ TransitionProps }) => (
+                            <Fade {...TransitionProps} timeout={350}>
+                                <ImagePreview src={asset.fileUrl || undefined} id="preview" />
```

Comment:
Why do you need this id? 
"preview" is probably too generic since it must be unique, also this component can be rendered multiple times on the same page which would cause this id to exist multiple times.